index.js

```diff
@@ -53,36 +77,48 @@
 
 
 // ***** Question 5 *****
+function updateGrade(student, grade){
+  student.grade = grade
+}
+
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 5", "color: red")
-// const student1 = { name: "Duane", grade: 88 }
-// updateGrade(student1, 92)
-// console.log(student1)
+ console.log("%cQuestion 5", "color: red")
+ const student1 = { name: "Duane", grade: 88 }
+ updateGrade(student1, 92)
+ console.log(student1)
 // // => { name: "Duane", grade: 92 }
-// console.log("%c----------", "color: red")
+ console.log("%c----------", "color: red")
```

Comment:
Nice work upgrading the student grade using object dot notation. 

A Thought Exercise: Why were we able to make a "change" to an object created using const?